src/middleware/accessLogger.js

```diff
@@ -0,0 +1,28 @@
+import morgan from 'morgan';
+import logger from '../utils/logger.js';
+import { getConfig } from '../config.js';
+
+const stream = {
+  write: (message) => logger.http(message)
+};
+
+const accessLogger = () => {
+  const { enableAccessLogging } = getConfig();
```

Comment:
@jchartrand I noticed that you are referencing `enableAccessLogger` as a config variable. However, there is no definition of the corresponding environment variable in `README.md`. You mentioned that the presence of `ERROR_LOG_FILE` and `ALL_LOG_FILE` are used to determine if error/info logging, but you did not define a similar variable for access logging. Considering that this variable is being used to control access logging, I have added the corresponding environment variable.